index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/695f3209b8.js" crossorigin="anonymous"></script>
+    <title>Portfolio Desktop</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="nav-left">
+                <span>Atilla Taha</span>
+            </div>
+            <div class="nav-right">
+                <ul>
+                    <li><a href="#">Portfolio</a></li>
+                    <li> <a href="#">About</a> </li>
+                    <li> <a href="#">Contact</a> </li>
+                    <li class="fas fa-envelope-square"></li>
+                </ul>
+            </div>
```

Comment:
Please align the navigation items center vertically. Kindly see the screenshot below for reference.
![mob-nav](https://user-images.githubusercontent.com/35504149/135470981-b22f177d-87d3-4581-bb15-ed67c8f2c9d4.png)